recipe/meta.yaml

```diff
@@ -1,14 +1,13 @@
-{% set name = "xcape" %}
 {% set version = "0.1" %}
-{% set sha256 = "288dd48bf9257cf7afed92c45b6c98b50db60e5cd13cd4d964e2522c6add84bd" %}
 
 package:
-  name: {{ name|lower }}
+  name: xcape
   version: {{ version }}
 
 source:
-  url: https://github.com/xgcm/xcape/archive/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/ocefpaf/xcape.git
+  git_tag: use_find_packages
```

Comment:
Do not merge this. If it works let's wait for a new release then we can update this PR.

Reply:
Is this the secret sauce that fixes #3?

Is this not something that should be fixed in our `setup.py` instead?

Reply:
That is the fix in upstream's setup.py. I sent a PR with it and we should wait for a new release before updating and merging this one.

I only built from my fork to ensure it would work here too b/c upstream only test with editable installs.